back/config/locales/en.yml

```diff
@@ -48,6 +48,14 @@ en:
     implemented_description: This idea has been implemented by the city
     rejected: rejected
     rejected_description: This idea has been rejected
+    expired: expired
```

Comment:
Is Crowdin smart enough to suggest the correct translations? If not, then I could also prepare the correct translation files and bulk upload them (there are some issues with bulk uploading in Crowdin though)?

Reply:
I think it is, but proofreaders will have to approve.

Crowdin uses something they call "Translation memory", where they will translate the same source strings with the same translation as in the past.

Reply:
Only waiting for French, Maori and Greenlandic right now, however some locales (Serbian, Romanian, Portuguese) are no longer supported in Crowdin. Perhaps we can directly edit the translation files for those locales?